### DISCUSSION TITLE
Show all (incl. unopened) workspaces on i3

### DISCUSSION THREAD

**Discussion by nullbyto** [polybar/polybar#2772] [Ideas]
Hello,

Is there any way to keep all the unopened workspaces in `i3` shown without having a window in it?
I know `i3` is per design not showing non created workspaces.
But maybe `Polybar` can do that. As the maintainers of `i3` are not interested in that.

**Benefit:**
- For looks.
- Consistency in the bar not moving on the left side.
- To see what numbers/icon is which has before switching, for some people this is good as they tend to rely on visuals rather than memory.

It could be added as just to show them without actually opening them. Only using `Polybar` which doesn't rely on i3.

**For example in the `i3` module:**
- To have a setter flag for that of course.
- To set which workspaces to show with its name/icon.

To distinguish between unopened and opened ones, an extra background color can be set for unopened workspaces.

I would like to see some feedback on that and opinions.

**Comment by patrick96** [comment]
We have long ago decided not to add such a feature (see https://github.com/polybar/polybar/issues/274#issuecomment-379365843).
